src/index.html

```diff
@@ -5,10 +5,30 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo-header">
+      <a href="#home" class="header-logo"></a>
+      <nav class="header-nav">
+        <ul class="header-list">
+          <li class="header-item"><a href="#" class="header-item__link is-active">Apple</a></li>
+          <li class="header-item"><a href="#" class="header-item__link">Samsung</a></li>
+          <li class="header-item"><a href="#" class="header-item__link">Смартфоны и телефоны</a></li>
+          <li class="header-item" data-qa="hover">
+            <a href="#" 
+              class="header-item__link" 
+              data-qa="hover">Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header-item"><a href="#" class="header-item__link">Гаджеты</a></li>
```

Comment:
```suggestion
          <li class="header-item">
            <a href="#" class="header-item__link">Гаджеты</a>
          </li>
```
Один тег - одна линия